src/mobility/components/CarSharingStationBottomSheet.tsx

```diff
@@ -57,6 +58,28 @@ export const CarSharingStationSheet = ({stationId, distance, close}: Props) => {
   const isLoading = isLoadingStation || isLoadingBenefit;
   const isError = isLoadingError || isBenefitError;
 
+  const previewCarIconCount = station ? station.capacity : 0;
+
+  /** 
+   *  carPreview function
+   * 
+   *  this function returns an array of vehicles that is shown
+   *  on the bottom sheet for car station mobility.
+   * 
+   *  Case can be accessed in Figma link below 
+   *  https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App?node-id=20471-12079
+   * 
+   *  Case on version 1.44 : 
+   *  - If the station capacity is 2 or less : show car image(s)
+   *  - If the station capacity is 3 or more : show 1 car image and the plus text
+   */
+
+  function carPreview(
```

Comment:
A somewhat confusing name for this function. The comment explains it, but rather trivial code as this should be made self explanatory. Could it be a solution to extract the entire car preview stuff into a separate component? `<CarPreview typesAvailable={vehicleTypesAvailable} />`

Reply:
Updated, moved the function to a CarPreviews component on file CarPreviews.tsx